wear/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "com.eveningoutpost.dexdrip"
+        minSdkVersion 20
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0.2"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.ustwo.android:clockwise-wearable:1.0.2'
+    compile 'com.google.android.support:wearable:1.1.0'
+    compile 'com.google.android.gms:play-services-wearable:7.3.0'
```

Comment:
This is how they were in NightScout and in this combination and they work. (I tried newer versions, but they didn't on my phone a couple of weeks ago. The cheap chinese 4.4 watches and miniphones can't update to a newer version.)

The only thing I changed was to restrict the play services to "wearable" to not have problems with the DEX method limit.